code/modules/tgui_panel/stat.dm

```diff
@@ -23,8 +23,6 @@
  * Sets the different available tabs.
  */
 /datum/tgui_panel/proc/set_tab_info(payload)
-	if(!is_ready())
```

Comment:
tgui now inits AFTER the client's new() so if this is still there, no data would be sent to the stats panel and it would be empty until everything and SSstats inits. plus these messages gets added to a queue anyways